custom_report_label/models/custom_report_label.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class CustomReportLabel(models.Model):
+    _name = "custom.report.label"
+
+    name = fields.Char(required=True)
+    model_id = fields.Selection(
+        selection="_list_all_models", string="Model", required=True
+    )
+    template_id = fields.Many2one("ir.actions.report")
+    existing_label = fields.Char(
```

Comment:
Why not make it include >< by default instead of including >< here?

I mean when the user types "Description" it replaces ">Descrption<" in report?.